action/staking/stake.go

```diff
@@ -92,6 +92,11 @@ func (s stakeTx) Validate(ctx *action.Context, tx action.SignedTx) (bool, error)
 		return false, err
 	}
 
+	val, err := ctx.Validators.Get(st.ValidatorAddress)
```

Comment:
in the validate function, we should not read from store, validate only validate tx by the data itself